src/templates/static/home.jsx

```diff
@@ -403,6 +403,12 @@ const Home = () => {
                             </div>
                         </div>
                     </div>
+                    <SeparatorLine className='gr-bottom-30' show_mobile />
+                </div>
+            </div>
+            <div >
+                <div className='trustpilot-widget .gr-centered-m' data-locale='en-US' data-template-id='53aa8912dec7e10d38f59f36' data-businessunit-id='58dc92220000ff00059f7dc4' data-style-height='150px' data-style-width='100%' data-theme='light' data-stars='4,5' data-review-languages='en'>
```

Comment:
typo? `.gr-centered-m`

Reply:
checking

Reply:
used to grantee centered margin, removed as being in center handled by other way

Reply:
@behnaz-fs that's not what I meant, in beginning of class names we don't use `.` in html, only in css

Reply:
ohh you are right, dot was extra, no need to this class anymore